tests/SOLID/LiskovSubstitutionPrinciple/Trial/BranchOfficeTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Deg540\PHPTestingBoilerplate\Test\SOLID\LiskovSubstitutionPrinciple\Trial;
+
+use Deg540\PHPTestingBoilerplate\SOLID\LiskovSubstitutionPrinciple\Trial\BackEndDeveloper;
+use Deg540\PHPTestingBoilerplate\SOLID\LiskovSubstitutionPrinciple\Trial\BranchOffice;
+use PHPUnit\Framework\TestCase;
+
+class BranchOfficeTest extends TestCase
+{
+    private BranchOffice $branchOffice;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $backEndDeveloper = new BackEndDeveloper(
+            'Javier',
+            'javier@gmail.com',
```

Comment:
¿ponemos los email de 540? Para este caso y los demás. 
```suggestion
            'javier@540deg.com',
```

Reply:
Cambaido